snapcraft/hooks/disconnect-plug-ovn-run

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+if [ "${openvswitch_builtin:-"false"}" = "true" ]; then
+    export OVS_RUNDIR="${SNAP_COMMON}/openvswitch/run/"
+    (
+        set -e
```

Comment:
I'm probably missing something but I couldn't find an explanation for having `OVS_RUNDIR` set outside that subshell? In fact, I don't understand the purpose of those 2 subshells. If you could shed some light I'd appreciate, thanks.